config/prod/usage-statistics-S3.yaml

```diff
@@ -10,7 +10,6 @@ parameters:
   OwnerEmail: 'larsson.omberg@sagebase.org'
```

Comment:
should this change to @tthyer ?

Reply:
No, let's leave him as owner of that (it ain't broke)
I'll make a second pass on this and fix some things up in a separate PR